src/kfactory/pcells/__init__.py

```diff
@@ -1,2 +1,28 @@
 # flake8: noqa
-from . import bezier, circular, dbu, euler, taper, waveguide
+from typing import Callable
+
+import kfactory as kf
+
+from .bezier import bend_s
+from .circular import bend_circular
+from .euler import bend_euler, bend_s_euler
+from .taper import taper as taper_function
+from .waveguide import waveguide as wg
+
+__all__ = [
+    "bend_s",
+    "bend_circular",
+    "bend_euler",
+    "bend_s_euler",
+    "taper",
+    "waveguide",
+]
+
+pcells: dict[str, Callable[..., kf.kcell.KCell]] = {  # type: ignore
+    "bend_s": bend_s,
+    "bend_circular": bend_circular,
+    "bend_euler": bend_euler,
+    "bend_s_euler": bend_s_euler,
+    "taper": taper_function,
+    "waveguide": wg,
+}
```

Comment:
why?

Reply:
This is used by get_component to get KCells from the .pcells